gabarit/template_num/num_project/tests/test_model_dense_classifier.py

```diff
@@ -39,6 +41,15 @@
 def remove_dir(path):
     if os.path.isdir(path): shutil.rmtree(path)
 
+def compare_keras_models(model1: Model, model2: Model) -> bool:
```

Comment:
No need to put type hinting in the tests. Delete them in order to also delete the import of Model from tensorflow (l30)